packages/strapi/lib/core/load-middlewares.js

```diff
@@ -17,18 +17,16 @@ module.exports = async function(strapi) {
 
   const loaders = createLoaders(strapi);
 
-  await Promise.all([
-    // load installed middlewares
-    loaders.loadMiddlewareDependencies(installedMiddlewares, middlewares),
-    // internal middlewares
-    loaders.loadInternalMiddlexares(middlewares),
-    // local middleware
-    loaders.loadLocalMiddlewares(appPath, middlewares),
-    // plugins middlewares
-    loaders.loadPluginsMiddlewares(installedPlugins, middlewares),
-    // local plugin middlewares
-    loaders.loadLocalPluginsMiddlewares(appPath, middlewares),
-  ]);
+  // load installed middlewares
+  await loaders.loadMiddlewareDependencies(installedMiddlewares, middlewares);
+  // internal middlewares
+  await loaders.loadInternalMiddlexares(middlewares);
```

Comment:
Since we modify this code, can you fix the typo please?

Reply:
@petersg83 I fixed in the RBAC PR I won't do it here to avoid conflicts later on :)